NEWS.md

```diff
@@ -21,7 +21,17 @@ OpenSSL 3.1
 
 ### Major changes between OpenSSL 3.1.1 and OpenSSL 3.1.2 [under development]
 
-  * none
+  * When building with the `enable-fips` option, the FIPS provider is
+    installed to ensure FIPS 140-3 compliance by default.  This
+    installation has two notable side effects:
+
+    1. TLS 1.2 necessitates use of an extended master secret (EMS).
+       This could lead to interoperability issues with older devices
+       not supporting TLS 1.3 or EMS in TLS 1.2.  This is as per the
+       FIPS 140-3 IG G.Q transition.
+
+    2. Truncated digests cannot be utilised with the Hash and HMAC DRBGs.
+       This is as per the FIPS 140-3 IG G.Q transition.
```

Comment:
I would expect the NEWS.md entry being more terse than the CHANGES.md entry and not the other way around. This also sounds like just installation enables the FIPS provider where it is necessary to also modify the config file or load the provider from an application explicitly.